docs/src/methods.md

```diff
@@ -0,0 +1,15 @@
+# Methods
+
+## Confidence interval
+```@docs
+confint
+confint(::BinomialTest)
```

Comment:
I don't think you need to list the specific methods. IIRC only methods from this package will appear.

Reply:
Actually, I do have to list the specific methods as we defined the general docstring on a zero-method function. If I just list `confint`, it will not include the docstrings for the specific methods in the documentation.

I actually like it this way as it gives us flexibility. We could potentially rearrange the method section or move the specific confidence intervals or p-values to the respective tests.

Note that in the REPL, `?confint` still gives you all methods without having to specify them in the help call.

Reply:
OK, I always forget how this works.